File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var removeDuplicates = function(nums) {
    let k=0
    dup=[]
    for(let i=0;i<nums.length;i++){
        if (dup.includes(nums[i])==false){
            nums[k]=nums[i]
            dup.push(nums[i])
            k++
        }    
    }
    return k
};

File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        int first=nums[0];
        vector<int> ret;
        ret.push_back(first);
        for(size_t i=1;i<nums.size();i++){
            if(nums[i-1] !=nums[i]){
                ret.push_back(nums[i]);
                first=nums[i];
            } 
        }
        nums=ret;
        return ret.size();
    }
};